src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -154,38 +156,31 @@ public void actionPerformed(ActionEvent e) {
 
         EasyBind.subscribe(searchField.textProperty(), searchText -> performSearch());
 
-        /*
-        String endSearch = "endSearch";
-        searchField.getInputMap().put(Globals.getKeyPrefs().getKey(KeyBinding.CLEAR_SEARCH), endSearch);
```

Comment:
Ah I thought the commented code ws just obsolete. I will see if I can fix this issue, too